BookingsApi/BookingsApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -75,6 +75,43 @@ public async Task<JusticeUser> SeedJusticeUser(string userName, string firstName
 
             return justiceUser.Entity;
         }
+        
+        public async Task<List<JusticeUser>> SeedJusticeUserList(string userName, string firstName, string lastName, 
+            bool isTeamLead = false, bool hasAllocated = false)
+        {
+            await using var db = new BookingsDbContext(_dbContextOptions);
+
+            var userList = new List<JusticeUser> { };
+            for (int i = 0; i < 10; i++)
+            {
+                var justiceUser = db.JusticeUsers.Add(new JusticeUser
+                            {
+                                ContactEmail = userName + i,
+                                Username = userName + i,
+                                UserRoleId = isTeamLead ? (int)UserRoleId.VhTeamLead : (int)UserRoleId.Vho,
+                                CreatedBy = $"integration{i}.test@test.com",
+                                CreatedDate = DateTime.Now,
+                                FirstName = firstName + i,
+                                Lastname = lastName + i,
+                            });
+
+                if (i == 9 && hasAllocated)
```

Comment:
What does the 9 here represent? If it's a role can you assign this to a named variable so it's clear what 9 represents?

Reply:
this is only to pick up the last in the list and make it allocated